### PR TITLE
add ppp and image

### PR DESCRIPTION
**Pull request by jerrrren**

### CODE REVIEW

File: docs/team/jerrrren.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+layout: page
+title: John Doe's Project Portfolio Page
```
Comment (kennethk-1201) [inline docs/team/jerrrren.md:3]: Update to 'Jer En's Project Portfolio Page'

File: docs/team/jerrrren.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+### Project: AddressBook Level 3
```
Comment (kennethk-1201) [inline docs/team/jerrrren.md:6]: Update to 'Project: SudoHR'

**Review by kennethk-1201 — APPROVED** [review]
Just update your doc based on the comments then can merge already

File: docs/team/jerrrren.md (hunk 2 above)
Comment (4ndrelim) [inline docs/team/jerrrren.md:6]: Yup, do make the requested changes before merging.

**Review by 4ndrelim — CHANGES_REQUESTED** [review]

File: docs/team/jerrrren.md (hunk 1 above)
Comment (jerrrren) [reply docs/team/jerrrren.md]: I have rectified this

File: docs/team/jerrrren.md (hunk 2 above)
Comment (jerrrren) [reply docs/team/jerrrren.md]: I have rectified this

**Review by 4ndrelim — APPROVED** [review]
LGTM

**Pull request merged**